### ISSUE TITLE
`openWindowSwitch: true` allows enable/disable feature but not trigger state

### ISSUE THREAD

**Issue by distantnative** [SeydX/homebridge-tado-platform#107]
When I add `openWindowSwitch: true` to a zone, I get a new switch that allows me to enable/disable the open window detection feature for the room, however it seems like it doesn't allow me to trigger an open window state (or I am just too blind to figure out how).

My use case would be that a contact sensor would be the trigger for an automation that sets the state in todo (as tado's own detection doesn't work reliably for me).

Thanks a lot for the great plugin!

**Comment by jotzet79** [comment]
Hi,
I also wanted to configure my HB & HK Setup for an automated trigger after the open window sensor reacts. Unfortunately I bumped into the same issue as you. The HomeKit exposed switches are only for enabling the feature of openwindow detection, but not the triggering of the special temporal temperature mode. 

I ended up implementing my own solution based on CmdTrigger and calling BASH script (curl), after the sensor fires. Works reliably since days… If interested I can publish it on GitHub of course.

PS also opened an issue some months ago #37 - but this was V1.

**Comment by distantnative** [comment]
@jotzet79 would be great if you could share your solution

**Comment by jotzet79** [comment]
@distantnative what makes me a bit wondering is that you would like that a _physical_ open window sensor triggers the OpenWindow mode via Tado, correct?

please bare in mind that this is ONLY possible if the Tado Service detects the window being opened (via temp drop and humidity change). Only then the Heating can be disabled for the specific time. 

So in your case you would only need to implement an automation (using Fake Switches with a time value set), which sets the temperature to a lower value and after a certain time you set it to normal again?

Regards,
Joachim

**Comment by distantnative** [comment]
No, I'll use actual contact sensors on the windows. But indeed, I was so focused on triggering the Tado feature. But I could just set to a lower temperature in HomeKit on open and back to auto on close. And deactivate tado's feature.

**Comment by xyz667** [comment]
> No, I'll use actual contact sensors on the windows. But indeed, I was so focused on triggering the Tado feature. But I could just set to a lower temperature in HomeKit on open and back to auto on close. And deactivate tado's feature.

Thats the way to go. Forget tado‘s useless window open functionality.
I have HomeKit automations that turn the thermostats off on window open events and turn them back to „auto“ mode when windows are closed. This is working quite well in my house.

**Comment by distantnative** [comment]
Using contact sensors and Home to create my own open windows automations works great!

One last thing: I was considering to still use `openWindowSwitch: true` to let Todo know that a window was opened, so it shows it correctly in the Air comfort report. However, that does't seem to work. The switch seems to only activate the Tado detection, but not actually triggering Tado to  recognise an open window. Or is there any way to realise that?
